src/module/dice/STARoller.js

```diff
@@ -246,10 +251,26 @@ new Dialog({
     html.find('.form-group').css({'width': '375px',});
 
     const token = canvas.tokens.controlled[0];
-    if (!token) {
-      ui.notifications.error("Please select a token for the NPC Ship.");
-      return;
-    }
+
+if (!token) {
+    let systemsHtml = `
```

Comment:
Needs a little more work:  If you have a token selected but it isn't a ship, the roller still doesn't work as I think intended.


Use case:
1. Select Character
2. Start roller dialogue.

Expected:
Since no ship is selected, form elements for system and department are available.

Observed:
No form elements provided.

Reply:
Tested locally, looks fixed now.